Test fails due to bad indent with
>       NameError: name 'InvalidException' is not defined

Author: Alexander Grund (TU Dresden)
diff --git a/test/cpython/v3_13/test_exceptions.py b/test/cpython/v3_13/test_exceptions.py
--- a/test/cpython/v3_13/test_exceptions.py
+++ b/test/cpython/v3_13/test_exceptions.py
@@ -407,8 +407,8 @@ class ExceptionTests(__TestCase):
             def __init__(self_):
                 raise RuntimeError("can't instantiate BadException")
 
-            class InvalidException:
-                pass
+        class InvalidException:
+            pass
 
         @unittest.skipIf(_testcapi is None, "requires _testcapi")
         def test_capi1():
